hardhat.config.ts

```diff
@@ -49,7 +53,7 @@ const config: HardhatUserConfig = {
   },
   starknet: {
     venv: ".venv",
-    network: "l2_testnet",
+    network: process.env.L2_NETWORK,
```

Comment:
I feel that this value is equal to `"l2_testnet"` locally and in the CI. When should `L2_NETWORK` be different than `"l2_testnet"`?

Reply:
I use it for deployment.